pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: cli_util
-version: 0.3.0-nullsafety.0
+version: 0.3.0-nullsafety.0-dev
```

Comment:
I think the goal is to publish this once it rolls through the sdk etc, so we likely don't need the `-dev` here?

Reply:
Sure, we could also drop the `-dev` and publish after the role - we wouldn't need to wait for a subsequent sync for a changelog and pubspec change, and it gives us more of a signal to verify things.

Reply:
Ya I just question if its worth doing this for every single package or not?

Not sure what precedent we want to set here, I can see it either way.

Reply:
Note that our existing guidelines do _not_ say to do this (they suggest doing what @pq already did and just waiting to publish until the internal validation happens)